packages/paypal-commerce-integration/src/paypal-commerce-constants.ts

```diff
@@ -0,0 +1,4 @@
+export const loadingIndicatorStyles = {
```

Comment:
```suggestion
export const LOADING_INDICATOR_STYLES  = {
```
Can we follow this format for a constant? Other than this, this rest looks good to me.

Reply:
Updated